hack/e2e.sh

```diff
@@ -119,6 +119,7 @@ export WORKLOAD_CONTROL_PLANE_ENDPOINT_IP
 echo "Acquired Workload Cluster Control Plane IP: $WORKLOAD_CONTROL_PLANE_ENDPOINT_IP"
 
 # save the docker image locally
+export LDFLAGS=""
```

Comment:
Could we add a comment here why we do this?

Note: I don't think this script gets used by anybody for local testing. It is very opinionated about the CI environment.

Reply:
Makes sense. I will add a comment here.